can-view-parser.js

```diff
@@ -271,12 +271,22 @@ var callAttrStart = function(state, curIndex, handler, rest){
 
 var callAttrEnd = function(state, curIndex, handler, rest){
 	if(state.valueStart !== undefined && state.valueStart < curIndex) {
-		handler.attrValue(rest.substring(state.valueStart, curIndex));
+		var quotedVal = rest.substring(state.valueStart - 1, curIndex + 2);
```

Comment:
can you move the `quotedVal` into the `steal-remove-start`/`steal-remove-end`?

Reply:
Done!